package.json

```diff
@@ -42,14 +42,17 @@
     "axios": "^0.19.0",
     "joi-browser": "^13.4.0",
     "jwt-decode": "^2.2.0",
+    "linkedin-login-for-react": "^1.2.3",
```

Comment:
I didn't find place, where you used this library. If you didn't use this library - remove it